cmd/earth/main.go

```diff
@@ -794,6 +802,8 @@ func (app *earthApp) before(context *cli.Context) error {
 
 	app.buildkitdSettings.DebuggerPort = app.cfg.Global.DebuggerPort
 	app.buildkitdSettings.RunDir = app.cfg.Global.RunPath
+	app.buildkitdSettings.AdditionalArgs = app.buildkitdAdditionalArgs.Value()
```

Comment:
HAve to bridge from the CLI framework slice to a legit slice; may as well shove it into the `before`